packages/liquidator/test/Liquidator.js

```diff
@@ -1272,185 +1272,10 @@ contract("Liquidator.js", function(accounts) {
-                // will extend even if withdraw progress is 0% (typically this would be set to 50% +)
-                defenseActivationPercent: 0
-              };
-              const liquidator = new Liquidator({
-                logger: spyLogger,
-                expiringMultiPartyClient: empClient,
-                gasEstimator,
-                votingContract: mockOracle.contract,
-                syntheticToken: syntheticToken.contract,
-                priceFeed: priceFeedMock,
-                account: accounts[0],
-                empProps,
-                liquidatorConfig
-              });
-              // sponsor1 creates a position with 120 units of collateral, creating 100 synthetic tokens.
-              await emp.create(
-                { rawValue: convertCollateral("120") },
-                { rawValue: convertSynthetic("100") },
-                { from: sponsor1 }
-              );
-
-              // liquidatorBot creates a position to have synthetic tokens to pay off debt upon liquidation.
-              // does not have enough to liquidate entire position
-              await emp.create(
-                { rawValue: convertCollateral("1000") },
-                { rawValue: convertSynthetic("90") },
-                { from: liquidatorBot }
-              );
-
-              // Start with a mocked price of 1 usd per token.
-              // This puts both sponsors over collateralized so no liquidations should occur.
-              priceFeedMock.setCurrentPrice(convertPrice("1"));
-              // sponsor withdraws below collat ratio
-              await emp.requestWithdrawal({ rawValue: convertCollateral("10") }, { from: sponsor1 });
-              await liquidator.update();
-              await liquidator.liquidatePositions();
-
-              // There should be exactly one liquidation in sponsor1's account. The liquidated collateral should be the original
-              // amount of collateral minus the collateral withdrawn. 120 - 10 = 110
-              let liquidationObject = (await emp.getLiquidations(sponsor1))[0];
-              assert.equal(liquidationObject.sponsor, sponsor1);
-              assert.equal(liquidationObject.liquidator, liquidatorBot);
-              assert.equal(liquidationObject.state, LiquidationStatesEnum.PRE_DISPUTE);
-              // minimum liquidation size / outstanding tokens * (outstanding collateral - withdraw amount)
-              // 5 / 100 * ( 120 -10) = 5.5
-              assert.equal(liquidationObject.liquidatedCollateral, convertCollateral("5.5"));
-              // minimum liquidation size / outstanding tokens * outstading collateral
-              // 5 / 100 * 120 = 6
-              assert.equal(liquidationObject.lockedCollateral, convertCollateral("6"));
-            }
-          );
-          versionedIt([{ contractType: "any", contractVersion: "any" }])(
-            "trigger multiple extensions and finally full liquidation",
```

Comment:
@daywiss This test is renamed and is able to check all possible cases in one:

- WDF is active
- Sponsor1 and sponsor2 create 100 token positions
- Liquidator starts with 150 tokens
- Sponsor1 and sponsor2 both request withdrawals
- Liquidator liquidates `sponsor1` who can be fully liquidated, bot has 50 tokens left
- Liquidator can only submit min liquidations on sponsor2, but not until the activation threshold has passed
- Finally, once sponsor2's withdrawal has expired, check that liquidator uses its full balance to liquidate sponsor2